src/editors/table.js

```diff
@@ -317,7 +317,7 @@ JSONEditor.defaults.editors.table = JSONEditor.defaults.editors.array.extend({
 
     // Buttons to delete row, move row up, and move row down
     if(!this.hide_delete_buttons) {
-      self.rows[i].delete_button = this.getButton('','delete','Delete');
```

Comment:
Just 'Delete' text, whereas the the array.js editor displays the item title too. Is this intentional? Nevertheless, I've provided distinct translation keys for both versions of the Delete button text.